src/tests/ftest/util/network_utils.py

```diff
@@ -16,7 +16,7 @@
 SUPPORTED_PROVIDERS = (
     "ofi+cxi",
     "ofi+verbs;ofi_rxm",
-    "ucx+dc_x",
+    "ucx+dc_mlx5",
```

Comment:
Note, this only adds `ucx+dc_mlx5` as a specifiable option.  If we want to change what is run by default in PRs, daily-testing, and provider-testing then the following Jenkinsfile changes will be required:
- (master, so this PR) https://github.com/daos-stack/daos/blob/master/Jenkinsfile#L1240
- (daily-testing, new PR after master PR lands) https://github.com/daos-stack/daos/blob/daily-testing/Jenkinsfile#L114
- (provider-testing, new PR after master PR lands) https://github.com/daos-stack/daos/blob/provider-testing/Jenkinsfile#L100